dbms/src/WindowFunctions/IWindowFunction.h

```diff
@@ -68,5 +68,8 @@ struct WindowFunctionWorkspace
     ColumnNumbers arguments;
 
     UInt64 idx;
+
+    // Avoid the release of column pointer
+    Columns materialized_columns;
```

Comment:
Should put it right after `argument_columns`

Reply:
> Should put it right after `argument_columns`

done